core/src/test/scala/integration/kafka/api/MetricsTest.scala

```diff
@@ -80,8 +80,8 @@ class MetricsTest extends IntegrationTestHarness with SaslSetup {
    * Verifies some of the metrics of producer, consumer as well as server.
    */
   @ParameterizedTest(name = "testMetrics with systemRemoteStorageEnabled: {1}")
```

Comment:
Please change this as following to fix error.
```suggestion
  @ParameterizedTest(name = "testMetrics with systemRemoteStorageEnabled: {0}")
```